README.md

```diff
@@ -2,557 +2,87 @@
-Request:
-```javascript
-{
-    name: Cameron Marks,
-    email: example@example.com,
-    password: password,
-    password_confirmation: password,
-    phone_number: 555-555-5555,
-    role: admin
-}
-```
-Response:
-```javascript
-{
-    id: 1,
-    name: Cameron Marks,
-    email: example@example.com,
-    password_digest: qp34uchon8w47yabov8ezfhpv,
-    phone_number: 555-555-5555,
-    role: admin,
-    active: true,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### PATCH Users /api/v1/users/:id
-Request:
-```javascript
-{
-    phone_number: 777-555-5555
-}
-```
-Response:
-```javascript
-{
-    id: 1,
-    name: Cameron Marks,
-    email: example@example.com,
-    password_digest: qp34uchon8w47yabov8ezfhpv,
-    phone_number: 777-555-5555,
-    active: true,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### GET ResourceTypes /api/v1/resource_types
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-```javascript
-[
-    {
-        id:1,
-        category: Bright Tank,
-        company: Tanks.com,
-        contact_number: 444-444-4444,
-        contact_name: John Johnson,
-        active: true,
-        created_at: 2019-07-16T21:52:00.823Z,
-        updated_at: 2019-07-16T21:52:00.823Z
-    },
-    {
-        id:2,
-        category: Van,
-        company: Chevrolet,
-        contact_number: 333-333-3333,
-        contact_name: John Johnsonite,
-        active: true,
-        created_at: 2019-07-16T21:52:00.823Z,
-        updated_at: 2019-07-16T21:52:00.823Z
-    }
-]
-```
-### GET ResourceTypes /api/v1/resource_types/:id
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-```javascript
-{
-    id:1,
-    category: Bright Tank,
-    company: Tanks.com,
-    contact_number: 444-444-4444,
-    contact_name: John Johnson,
-    manual_url: example.google.com
-    active: true,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### POST ResourceTypes /api/v1/resource_types
-Request:
-```javascript
-{
-    category: Bright Tank,
-    company: Tanks.com,
-    contact_number: 444-444-4444,
-    contact_name: John Johnson,
-    manual_url: example.google.com
-}
-```
-Response:
-```javascript
-{
-    id:1,
-    category: Bright Tank,
-    company: Tanks.com,
-    contact_number: 444-444-4444,
-    contact_name: John Johnson,
-    manual_url: example.google.com
-    active: true,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### PATCH ResourceTypes /api/v1/resource_types/:id
-Request:
+We modeled our apps with the help of Great Divide Brewery[https://greatdivide.com/]; after touring their facilities and discussing some of the dilemmas the face on a daily basis, we designed a solutions to as many as possible in one bundle.
```

Comment:
Spelling error: the dilemmas the face ==> the dilemmas they face

Reply:
I’m not sure but should “we designed a solutions to as many as possible” be “we designed solutions to as many as possible”?

Reply:
yes